mumlib/Cargo.toml

```diff
@@ -13,6 +13,7 @@ readme = "../README.md"
 
 [dependencies]
 colored = "2"
+chrono = "0.4"
```

Comment:
Could be worth considering if it's worth adding `chrono` as a dependency since it adds 8 new compilation tasks when building this project. For example, it might be possible to approximate the behavior that `chrono` gives us by using functions in `std`